packages/utilities/gel-styled-components/README.md

```diff
@@ -0,0 +1,29 @@
+# gel-styled-components &middot; [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/BBC-News/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/gel-styled-components.svg)](https://www.npmjs.com/package/@bbc/gel-styled-components) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://reactjs.org/docs/how-to-contribute.html#your-first-pull-request)
+
+This package is a collection of variable constants which can be imported into your application.
+
+## Installation
+
+```jsx
+npm install @bbc/gel-styled-components --save
+```
+
+## Usage
+
+```jsx
+import { GEL_BREVIER } from 'gel-styled-components/typography';
+
+import { GEL_GROUP_3_SCREEN_WIDTH_MIN } from 'gel-styled-components/breakpoints';
+```
+
+## Contributing
```

Comment:
You have a Contributing section here as well as in the `CONTRIBUTING.md`. They say pretty much the same thing. I don't mind which is kept, but we likely don't need both.